### PR TITLE
Set up Datawarehouse

### PR DESCRIPTION
**Pull request by happilyeverafter95**
To do: set up DB and populate credentials. 

Airflow jobs will start writing to these

### CODE REVIEW

File: db/tables.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    symbol = Column('stock_symbol', String)
+    price = Column('price', Float)
+    transaction_type = Column('transaction_type', String)
+
+
+engine = DB.create_engine()
+
+# create schema if doesn't exist
+if not engine.dialect.has_schema(engine, settings.env):
+    engine.execute(schema.CreateSchema(settings.env))
```
Comment (edwardcqian) [inline db/tables.py:24]: What does the `settings.env` do here?

**Review by edwardcqian — APPROVED** [review]
Small question otherwise LGTM

File: db/tables.py (hunk 1 above)
Comment (happilyeverafter95) [reply db/tables.py:24]: Schema is based on env. I was thinking having a test environment and a prod one. What you think?

Comment (edwardcqian) [reply db/tables.py:24]: Sounds good

Comment (edwardcqian) [reply db/tables.py:24]: There is no default value for env. Would it being None cause an error?

Comment (happilyeverafter95) [reply db/tables.py:24]: yeah it would, but `settings.env` is populated from `.envrc` which should be set to `development`

**Pull request merged**